### ISSUE TITLE
Controller Issues in 'Multiplayer > Battle'

### ISSUE THREAD

**Issue by SecularSteve** [mmatyas/openblok#37]
The Rotation Buttons don't work in Battle Mode, is this intentional?

My controller is an Original PS3 Controller detected as a ShanWan PS3/PC Wired Gamepad (in case it's important).

**Comment by mmatyas** [comment]
I've just tried an 1v1 battle and it seems to be working fine for me (tested with Logitech F310 and a noname chinese SNES controller).

Are both gamepads PS3 controllers? Would it be possible that they interfere each other somehow?

**Comment by SecularSteve** [comment]
Sorry for not responding earlier.
No, I just used literally one controller (and chose one player). Is this some kind of unintended bug?

**Comment by mmatyas** [comment]
Yes, it does seem to be some controller related bug. Do you have any other gamepads to tests this?

**Comment by SecularSteve** [comment]
Nope (but I haven't tried it with Shibari yet).
And It's on Windows 10.
